lib/bot-service.js

```diff
@@ -4,7 +4,7 @@ const Store = require('./store');
 const https = require('https');
 const jwt = require('jsonwebtoken');
 
-const HOST = 'iot.bankingofthings.io';
+const HOST = 'iot-dev.bankingofthings.io';
```

Comment:
should put this back, only for local development

Reply:
Yes, retained this to test and then revert back.

Reply:
Tested the changes on RPI Z Device - Single Pair, Multi Pair with BLE and QR Code, get actions and trigger payments. Reverted HOST to point to iot.bot